src/Tribe/Field.php

```diff
@@ -702,6 +706,52 @@ public function license_key() {
 			return $field;
 		}
 
+		/**
+		 * Generate an image field.
+		 * 
+		 * @since TBD
+		 *
+		 * @return string The field.
+		 */
+		public function image() {
+
+			tribe( Settings::class )->maybe_load_image_field_assets();
+
+			$image_exists = ! empty( $this->value );
+			$upload_image_text = esc_html__( 'Select Image', 'tribe-common' );
+			$remove_image_text = esc_html__( 'Remove Image', 'tribe-common' );
+
+			// Add default fieldset attributes if none exist.
+			$image_fieldset_attributes = [
+				'data-select-image-text' => esc_html__( 'Select an image to use in your email headers', 'tribe-common' ),
```

Comment:
The message about the email headers should come from ET once we define the setting. For now, I'd leave a generic message to "Select an image".